src/endpoints/substrate-endpoint/services/genetic-analysis.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable, Logger } from "@nestjs/common";
+import { ElasticsearchService } from "@nestjs/elasticsearch";
+
+@Injectable()
+export class GeneticAnalysisService {
+  private readonly logger: Logger = new Logger(GeneticAnalysisService.name);
+  constructor(private readonly elasticSearchService: ElasticsearchService) {}
+
+  async getGeneticAnalysByTrackingId(genetic_analyst_tracking_id: string) {
+    let hitsGeneticAnalys = [];
+
+    try {
+      const geneticAnalysis = await this.elasticSearchService.search({
+        index: 'genetic-analysis',
+        body: {
+          query: {
+            match: {
+              genetic_analyst_tracking_id: {
+                query: genetic_analyst_tracking_id,
+              },
+            },
+          },
+        },
+      });
+      hitsGeneticAnalys = geneticAnalysis.body.hits.hits;
+    } catch (error) {
+      if (error?.body?.error?.type === 'index_not_found_exception') {
+        await this.logger.log(
+          `API "genetic-analysis/{tracking_id}": ${error.body.error.reason}`,
+        );
+      } else {
+        throw error;
+      }
```

Comment:
```
if (error?.body?.error?.type !== 'index_not_found_exception') {
    throw error;
}


await this.logger.log(
    `API "genetic-analysis/{tracking_id}": ${error.body.error.reason}`,
);
```

Reply:
Why has this been resolved? @JackyRahman it has not been changed yet.
![image](https://user-images.githubusercontent.com/37565691/153816185-00c24043-260f-40cc-9a71-eed95b5d68e8.png)
Is there a reason why the code wasn't changed?